rkt/api_service.go

```diff
@@ -393,65 +527,12 @@ func getBasicPod(p *pod) *v1alpha.Pod {
 				// See https://github.com/coreos/rkt/pull/2331#issuecomment-203540543
 				//
 				// TODO(yifan): Revisit this when using unified cgroup hierarchy.
-				pod.Cgroup = strings.TrimSuffix(cgroup, "/init.scope")
+				v1pod.Cgroup = strings.TrimSuffix(cgroup, "/init.scope")
```

Comment:
@euank yeah... I was thinking about such situation, but missed this field. Should probably cache an immutable object.